partition/androidboot.go

```diff
@@ -75,3 +76,13 @@ func (a *androidboot) SetBootVars(values map[string]string) error {
 	}
 	return env.Save()
 }
+
+func (a *androidboot) Reboot() error {
```

Comment:
don't hardcode the folder here but in dirs/dirs.go and follow the patterns there, then you can use  dirs.SetRootDir in tests to have an alternate root dir